minio/minio
# Add monthly license update job

## Description

If the cluster is registered with SUBNET, run a job every month to get the latest license from SUBNET.

## Motivation and Context

Automate license renewal

## How to test this PR?

- Create a custom account on subnet
- For the purpose of testing, change value of `licUpdateCycle` inside `license-update.go` to a short value, say 30 seconds
- Build and start minio cluster
- Register the cluster on the custom account using `mc license register {alias}`
- Check the license expiry using `mc license info {alias}` to verify that it is same as subscription end date of the account
- Log in as engineer admin in subnet and extend the subscription of the account by a month (this renews the account's licenses on subnet side with extended expiry)
- Wait for a duration of more than `licUpdateCycle`
- Check the license expiry on minio side now using `mc license info {alias}` and verify that it has also got extended correctly

## Types of changes

## Checklist:
